### PR TITLE
Init Kojira CRD

### LINKED ISSUE #12: Init CRD for kojira
This issue is for creating initial structure of [CRD](https://docs.openshift.com/container-platform/4.3/operators/crds/crd-extending-api-with-crds.html) for [kojira component](https://github.com/fedora-infra/mbbox/tree/master/components/kojira).

## Acceptance criteria

* Create initial structure of CRD
* Create initial test files for component
* Create empty role

## Definition of Done

* Acceptance criteria are met
* Local tests have been run and pass - if failed it is re-opened/fixed and passed

### PR DESCRIPTION
**Pull request by Zlopez**
* CRD files were created by `operator-sdk` command
* asserts contain only one noop assert, which does nothing
* Kojira role contains just empty files and placeholders
* CRD is created for tests
* Add Kojira to watches

Closes #12

Signed-off-by: Michal Konečný <mkonecny@redhat.com>

### CODE REVIEW

**Comment by Zlopez** [comment]
The tests are now working, only thing that was missing was entry in `watches.yml` and creation of kojira CRD.

**Comment by Zlopez** [comment]
.recheck

**Comment by Zlopez** [comment]
recheck

**Comment by Zlopez** [comment]
recheck

**Comment by Zlopez** [comment]
recheck

**Comment by Zlopez** [comment]
recheck

**Comment by AdamSaleh** [comment]
recheck

File: mbox-operator/molecule/default/prepare/operator-resources.yml [hunk 1]
```diff
@@ -6,10 +6,6 @@
   vars:
     ansible_python_interpreter: '{{ ansible_playbook_python }}'
     deploy_dir: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') }}/deploy"
-    crd_paths:
```
Comment (AdamSaleh) [inline mbox-operator/molecule/default/prepare/operator-resources.yml:9]: Thanks! Forgot to clean this up :-)

**Pull request merged**